src/sql/src/parsers/alter_parser.rs

```diff
@@ -73,6 +73,26 @@ impl<'a> ParserContext<'a> {
                     parser.peek_token()
                 )));
             }
+        } else if parser
+            .expect_keywords(&[Keyword::ALTER, Keyword::COLUMN])
```

Comment:
+1 for "modify column" for the conciseness

Reply:
Even if you want to use `ModifyColumn`, you need to upgrade sqlparser-rs to the latest version first. There is no `Keyword::Modify` now.

Reply:
Maybe we could change the syntax or support `modify` in another PR.